src/style/mixins.scss

```
@@ -159,3 +159,13 @@
     }
   }
 }
+/// Mixin allows scroll snapping to a specific class if .iui-scroll-snapping is applied to the parent
+@mixin iui-scroll-snapping($selector: '> *') {
+  .iui-scroll-snapping {
```

I think it would be more useful if this class was applied on the same level instead of as a child, i.e.
```suggestion
&.iui-scroll-snapping {
```


However, that would make this mixin unusable at the root level, so we can take it one step further and make it conditional. The suggestion below will apply it as a modifier if it's used inside another class, otherwise it will leave it at root level.
```suggestion
#{if(&, '&.iui-scroll-snapping', '.iui-scroll-snapping')} {
```

---

Applied these changes, good suggestions.